openfisca_us/variables/gov/states/md/tax/income/credits/md_poverty_line_credit.py

```diff
@@ -0,0 +1,56 @@
+from openfisca_us.model_api import *
+
+
+class md_poverty_line_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD Poverty Line Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://law.justia.com/codes/maryland/2021/tax-general/title-10/subtitle-7/section-10-709/"
+
+    def formula(tax_unit, period, parameters):
+        # Get the federal poverty line for the tax_unit
+        # Find the number of persons in your family/household from the chart that is the same as the number of persons entered on your federal tax return. Enter the income level that corresponds to the number of person
+        fpg = tax_unit("tax_unit_fpg", period)
+        # Get the tax unit's earned income
+        earned_income = tax_unit("tax_unit_earned_income", period)
+        # Get total of md_agi and md_total_additions
+        # From the law:
+        # (2)    “Applicable poverty income level” means the amount specified in the poverty income standard that corresponds to the number of exemptions which the individual is allowed and claims under § 10–211(b)(1) of this title.
+
+        # (3)    “Eligible low income taxpayer” means an individual, or an individual and the individual’s spouse if they file a joint income tax return:
+        #     (i)    whose federal adjusted gross income as modified under §§ 10–204 through 10–206 of this title does not exceed the applicable poverty income level;
+        agi_plus_md_additions = add(
+            tax_unit,
+            period,
+            [
+                "adjusted_gross_income",
+                "md_total_additions",
+            ],
+        )
+        countable_income = max_(agi_plus_md_additions, earned_income)
```

Comment:
The law states this countable income only determines eligibility, while the credit value is based on earned income only. Does the tax form say differently? Could you cite it if you're using it?

Reply:
I think you're right - it's only for eligibility.

Here is the worksheet from the instuctions:
![image](https://user-images.githubusercontent.com/16396180/179290220-58d9657c-77c1-4e7e-8b7b-5b090b6fe85c.png)

And here is line 7:
![image](https://user-images.githubusercontent.com/16396180/179290468-83d36a57-7e56-47ba-a0a5-b9c4bb92135e.png)